docs/releases/openmathinstruct2/training.md

```diff
@@ -49,73 +49,66 @@ python -m nemo_skills.training.prepare_data \
 
 ```bash
-ns train \
+ns nemo_rl sft \
     --cluster=slurm \
     --expname=openmathinstruct2-repro-8b \
     --output_dir=/workspace/openmathinstruct2-repro/checkpoints \
-    --nemo_model=/workspace/llama3.1-8b-nemo \
+    --hf_model=/workspace/Llama-3.1-8B  \
     --num_nodes=8 \
     --num_gpus=8 \
-    --average_steps=10000,20000,30000,40000,50000,60000 \
+    --backend=megatron \
     --training_data=/workspace/openmathinstruct2-sft.jsonl \
-    ++model.data.train_ds.micro_batch_size=8 \
-    ++model.tensor_model_parallel_size=4 \
-    ++model.pipeline_model_parallel_size=1 \
-    ++model.optim.lr=2e-5 \
-    ++trainer.sft.save_interval=10000 \
-    ++trainer.sft.max_steps=60000 \
-    ++trainer.sft.max_epochs=100
+    ++policy.train_micro_batch_size=8 \
+    ++policy.train_global_batch_size=512 \
+    ++policy.tensor_model_parallel_size=4 \
+    ++policy.pipeline_model_parallel_size=1 \
+    ++policy.lr=2e-5 \
+    ++checkpointing.save_period=10000 \
+    ++sft.max_num_steps=60000 \
+    ++sft.max_num_epochs=100 \
+    ++policy.sequence_packing.enabled=False \
+    ++checkpointing.checkpoint_must_save_by=00:03:45:00
 ```
 
 For 70B model, we used 5M data subset and the following parameters, but training
 it longer is likely going to improve results.
 
+
 ```bash
-ns train \
+ns nemo_rl sft \
     --cluster=slurm \
     --expname=openmathinstruct2-repro-70b \
-    --output_dir=/workspace/openmathinstruct2-repro-70b/checkpoints \
-    --nemo_model=/workspace/llama3.1-70b-nemo \
+     --output_dir=/workspace/openmathinstruct2-repro-70b/checkpoints \
+    --hf_model=/workspace/Llama-3.1-70B  \
     --num_nodes=32 \
     --num_gpus=8 \
-    --average_steps=3330,6660,9990,13320,16650,20000 \
+    --backend=megatron \
     --training_data=/workspace/openmathinstruct2-sft-5M.jsonl \
-    ++model.data.train_ds.micro_batch_size=1 \
-    ++model.tensor_model_parallel_size=8 \
-    ++model.pipeline_model_parallel_size=2 \
-    ++model.optim.lr=1e-5 \
-    ++trainer.sft.save_interval=3330 \
-    ++trainer.sft.max_steps=20000 \
-    ++trainer.sft.max_epochs=100
+    ++policy.train_micro_batch_size=1 \
+    ++policy.train_global_batch_size=512 \
+    ++policy.tensor_model_parallel_size=8 \
+    ++policy.pipeline_model_parallel_size=2 \
+    ++policy.lr=1e-5 \
+    ++checkpointing.save_period=3330 \
+    ++sft.max_num_steps=20000 \
+    ++sft.max_num_epochs=100 \
+    ++policy.sequence_packing.enabled=False \
+    ++checkpointing.checkpoint_must_save_by=00:03:20:00
 ```
 
+
 If you have a job timeout, it's necessary to set the maximum time per run to 40 minutes
 before the timeout to allow for the final checkpoint to be saved. E.g. if your timeout is 4 hours,
-add `++exp_manager.max_time_per_run=00:03:20:00`
+add `++checkpointing.checkpoint_must_save_by=00:03:20:00`
 
 
-If you want to follow up with checkpoint conversion and evaluation, see
+If you want to follow up with evaluation, see
```

Comment:
Please add a note here that in the original work we did checkpoint averaging, but it's not done in the current code and results might be a bit different. Can also say that we will add the support soon